tests/util/mod.rs

```diff
@@ -56,7 +56,8 @@ impl TestCase {
     }
 
     fn check_zip215(&self) -> Result<(), Report> {
-        use ed25519_zebra_zip215::{Signature, VerificationKey};
+        use ed25519::Signature;
+        use ed25519_zebra_zip215::VerificationKey;
```

Comment:
With the re-export above we should be able to restore the original import

```suggestion
        use ed25519_zebra_zip215::{Signature, VerificationKey};
```